boston/README.md

```diff
@@ -144,17 +142,36 @@ gcloud ml-engine local train --module-name=trainer.task \
  --job-dir=$JOB_DIR
 ```
 
-*Feel free to modify the destination file for in utils.py
-
-* **Run in Google Cloud ML Engine:**
+* **Run in Google Cloud ML Engine**
 
 You can train the model on Cloud ML Engine:
 
+*NOTE:* If you downloaded the training files to your local filesystem, be sure
+to reset the `TRAIN_FILE` environment variable to refer to a GCS location.
+Data must be in GCS for cloud-based training.
+
+Run the code on Cloud ML Engine using `gcloud`. Note how `--job-dir` comes
+before `--` while training on the cloud and this is so that we can have
+different trial runs during Hyperparameter tuning.
+
+* **GCloud configuration:**
+
+```
+DATE=`date '+%Y%m%d_%H%M%S'`
+export JOB_NAME=census_$DATE
+export GCS_JOB_DIR=gs://your-bucket-name/path/to/my/jobs/$JOB_NAME
+echo $GCS_JOB_DIR
+export TRAIN_FILE=gs://cloud-samples-data/ml-engine/census/data/adult.data.csv
+export REGION=us-central1
+```
+
+* **Run in Google Cloud ML Engine:**
+
 ```
 gcloud ml-engine jobs submit training $JOB_NAME \
  --stream-logs \
  --runtime-version 1.10 \
- --job-dir=$JOB_DIR \
+ --job-dir=$GCS_JOB_DIR \
```

Comment:
perhaps consider keeping it as `JOB_DIR`, to be consistent with the name of the flag.